extensions/vscode/src/views/homeView.ts

```diff
@@ -765,62 +780,60 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
     this.requestWebviewSaveSelection();
   }
 
-  private async showSelectConfigForDeployment() {
-    const activeDeployment = this.getActiveContentRecord();
-    if (activeDeployment === undefined) {
+  private async showSelectOrCreateConfigForDeployment() {
+    const targetContentRecord = this.getActiveContentRecord();
+    if (targetContentRecord === undefined) {
       console.error(
-        "homeView::showSelectConfigForDeployment: No active deployment.",
+        "homeView::showSelectConfigForDeployment: No target deployment.",
       );
       return;
     }
-    const config = await selectConfig(activeDeployment, Views.HomeView);
-    if (config) {
-      const api = await useApi();
-      const apiRequest = api.contentRecords.patch(
-        activeDeployment.deploymentName,
-        config.configurationName,
-        activeDeployment.projectDir,
+    const targetConfiguration = this.getActiveConfig();
+    if (targetConfiguration === undefined) {
```

Comment:
This if condition change makes it so when you get into the state where there is a missing configuration nothing happens when clicking the `showSelectOrCreateConfigForDeployment`

Reply:
See code update: https://github.com/posit-dev/publisher/pull/1998/commits/9d9c3696d234cf30deaca5ccd32aa728f1debc03